lib/pure/os.nim

```diff
@@ -2288,8 +2288,8 @@ iterator walkDirRec*(dir: string,
   ## ---------------------   ---------------------------------------------
   ## followFilter            meaning
   ## ---------------------   ---------------------------------------------
-  ## ``pcDir``               follow real directories (default)
-  ## ``pcLinkToDir``         follow symbolic links to directories
+  ## `pcDir`               follow real directories (default)
+  ## `pcLinkToDir`         follow symbolic links to directories
```

Comment:
You may want to fix the indentation here.

Reply:
it has bugs when single backtick in markdown tables